bridge/setu/processor/checkpoint.go

```diff
@@ -109,6 +109,16 @@ func (cp *CheckpointProcessor) startPollingForNoAck(ctx context.Context, interva
 // 2. check if checkpoint has to be proposed for given headerblock
 // 3. if so, propose checkpoint to heimdall.
 func (cp *CheckpointProcessor) sendCheckpointToHeimdall(headerBlockStr string) (err error) {
+	status, err := helper.GetNodeStatus(cp.cliCtx)
```

Comment:
Why don't we use `cp.cliCtx.Client.Status()` to avoid another call to helper? I think you get the same result.

Reply:
`helper.GetNodeStatus()` is the call used all over the codebase so I think it's better to follow the convention.